app/javascript/mastodon/features/emoji/emoji_compressed.js

```diff
@@ -115,10 +115,12 @@ Object.keys(emojiIndex.emojis).forEach(key => {
 
 // JSON.parse/stringify is to emulate what @preval is doing and avoid any
 // inconsistent behavior in dev mode
-module.exports = JSON.parse(JSON.stringify([
+const ex = JSON.parse(JSON.stringify([
```

Comment:
```suggestion
const emoji = JSON.parse(JSON.stringify([
```

Would be a better name than `ex`